app/views/partials/_flash_messages.html.erb

```diff
@@ -0,0 +1,34 @@
+<% if flash[:error] %>
+<div class="window-outer error-outer row center-xs middle-xs mt-5">
+  <div class="col-md-6 col-sm-8 col-xs-12">
+    <div class="window">
+      <div class="title-bar">
+        <div class="title-bar-text">You broke something!</div>
+      </div>
+      <div class="window-body">
+        <p class="flex align-center">
+          <%= image_tag 'https://win98icons.alexmeub.com/icons/png/msg_error-0.png', class: 'mr-3' %>
+          <%= flash[:error] %>
+        </p>
+      </div>
+    </div>
+  </div>
+</div>
+<% end %>
+<% if flash[:notice] %>
+<div class="error-outer row center-xs middle-xs mt-5">
+  <div class="col-md-6 col-sm-8 col-xs-12">
+    <div class="window">
+      <div class="title-bar">
+        <div class="title-bar-text">Boo-yaaaaaaaaaa</div>
+      </div>
+      <div class="window-body">
+        <p class="flex align-center">
+          <%= image_tag 'https://win98icons.alexmeub.com/icons/png/trust0-0.png', class: 'mr-3' %>
+          <%= flash[:notice] %>
+        </p>
+      </div>
+    </div>
+  </div>
+</div>
+<% end %>
```

Comment:
- [ ] put the if statement in a partial that renders a partial containing the markup within the if statement so you don't have this complexity directly in the partial